src/fides/api/ops/service/connectors/fides/fides_client.py

```diff
@@ -41,7 +40,7 @@ def __init__(
         username: str,
         password: str,
     ):
-        self.session = Session()
+        self.session = Client(timeout=5.0)
```

Comment:
@ThomasLaPiana  i went ahead and pushed updates for this in https://github.com/ethyca/fides/pull/2246/commits/b19bf9753fb89e79ff2feaefe30221fc4d0643d0 - let me know what you think as a potential fix here.

this worked for my local smoke testing, and also passed the unit tests i could run locally (i was having trouble getting `nox -s "pytest(ops-integration)"` to spin up locally, seems like some issues with the msyql and timescale containers...)

Reply:
Thanks for the updates! Looks like one small problem but otherwise went off without a hitch :) 

Have you taken a look at the [timeout configuration](https://www.python-httpx.org/advanced/#timeout-configuration) docs here?

Seems we might be able to get tricky with it and set different types of timeouts. Personally, I think setting a 30 minute timeout across the board feels a bit strange, considering there is only a single request that has that outlier use-case.

I'm gonna see if we can set a timeout for that request specifically as higher than the default

Reply:
:) glad we arrived at the same place!